noxfile.py

```diff
@@ -5,4 +5,17 @@
 def test(session):
     session.install("-rrequirements-dev.txt")
     session.install("-e", ".", "--no-build-isolation")
+    session.run("pytest", "-m", "not lindera", *session.posargs)
+
+
+@nox.session(python=["3.8", "3.9", "3.10", "3.11", "3.12"])
+def test_lindera(session):
+    session.install("-rrequirements-dev.txt")
+    session.install(
+        "--no-build-isolation",
+        '--config-settings',
+        'build-args="--features=lindera"',
```

Comment:
It took a surprisingly long time to discover the correct way to spell how to get this value sent down to the build backend!

Reply:
Yes, even with the unified `pyproject.toml` approach, wiring Python builds is still a mess IMHO. Do you remember where in the Maturin documentation you would have expected this? Maybe an improvement is just a PR away?

Reply:
Maturin wasn't too bad, it is pretty easy to find "feature selection" and that shows up under the `build-wheels` section in the maturin docs. Much more difficult to find was how to propagate the feature parameter through the pip interface, as above.  

I had a quick look, I think this section is probably the best place to mention the `build-args` parameter: https://www.maturin.rs/index.html?highlight=pip%20install#python-packaging-basics